apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-notice.tsx

```diff
@@ -20,13 +20,12 @@ export const LivePreviewUpgradeNotice: FC< {
 	useHideTemplatePartHint();
 
 	const noticeText = sprintf(
-		// translators: %1$s: The previewing theme name, %2$s: The theme type ('WooCommerce' or 'Premium')
+		// translators: %s: plan name (e.g. Starter, Explorer).
 		__(
-			'You are previewing %1$s, a %2$s theme. You can try out your own style customizations, which will only be saved if you upgrade and activate this theme.',
```

Comment:
i'm not sure about getting rid of the "You can try out...." text, CC @Automattic/lego

Reply:
I thought it was not super relevant because we don't show a similar message for Free themes:

<img width="346" alt="Screenshot 2024-01-24 at 11 46 57" src="https://github.com/Automattic/wp-calypso/assets/1233880/51860130-fd6e-42cd-aba5-27f361e63a1c">

If we think that the try out aspect is important, I think we should also indicate it for Free themes (e.g. You can try out your own style customizations, which will only be saved if you activate this theme.).

Reply:
Ok, fair enough, lets move forward with this PR as-is then. The wording can always be added back later if it is necessary.

Reply:
> If we think that the try out aspect is important, I think we should also indicate it for Free themes

Actually the original idea is to do it for every plan. See the proposed design here: pekYwv-3uK-p2

Due to the complication in combining the logic with GS, which lives in different Calypso app, we decided to punt this.

For now I think the new wording proposed in this PR is better, so approving 👍